include/RAJA/policy/cuda/forall.hpp

```diff
@@ -627,7 +627,7 @@ forall_impl(resources::Cuda cuda_res,
                                                      IterationGetter,
                                                      Concretizer,
                                                      BlocksPerSM,
-                                                     Async> const&,
+                                                     Async> const& pol,
```

Comment:
Was the `pol` arg unused before?

Reply:
That's correct, we just passed through the policy type before, but Jason added a policy object/argument yesterday to simplify the type specification being passed around in the param reducer.